### ISSUE TITLE
Question about backward_G_alone

### ISSUE THREAD

**Issue by eveningglow** [junyanz/BicycleGAN#14]
In this [line](https://github.com/junyanz/BicycleGAN/blob/5bd76aca5ed3d9e62aebb578a28a1053b622ddf9/models/bicycle_gan_model.py#L143) could you explain why ` torch.mean(torch.abs(self.mu2 - self.z_random))` not ` torch.mean(torch.abs(self.z_predict - self.z_random))`?

**Comment by junyanz** [comment]
In the paper, we mentioned, "Note that the encoder E here is producing a point estimate for z, whereas the
encoder in the previous section was predicting a Gaussian distribution." (Sec 3.3). 
In practice, z_predict = Gaussian(mu2, std2). This loss might not be stable if your sigma is big.
